packages/frontend/src/contexts/game/reducer/index.js

```diff
@@ -6,6 +9,23 @@ export const gameReducer = (state, action) => {
   }
 
   switch (action.type) {
+    case types.ADD_TILES:
+      return {
+        ...state,
+        tiles: createTiles(action.payload.photos),
+      };
+    case types.FLIP_TILE:
```

Comment:
I wrote working tests for these this morning but I definitely think you should look at them. It looks like the second test for testing FLIP_TILE could just replace the first one and test everything in one scope since I am repeating the setup. 

I also started tests for the creatTilesFromPhotos helper, though I am still looking for a solution to check the results in the assertion. I think it's tricky because the results are an array of nested objects and toEquals() can't check for equality in a nested array of objects. 

I was thinking of turning both results into strings somehow and comparing those. 

I think everything we talked about should be done with this PR, other than tests for the helper function.